config/config.json

```diff
@@ -15,7 +15,9 @@
         "SessionLengthWebInDays": 30,
         "SessionLengthMobileInDays": 30,
         "SessionLengthSSOInDays": 30,
-        "SessionCacheInMinutes": 10
+        "SessionCacheInMinutes": 10,
```

Comment:
Ja, sorry I ran make test and forgot to revert config.json. submitting the fix now